.devcontainer/init-host.sh

```
@@ -0,0 +1,76 @@
+        cp -a "$HOME/.claude/.credentials.json" "$CLAUDE_STAGING/.credentials.json" 2>/dev/null || true
+    # Plugins: always copy (just IDs, no secrets)
+    if [ -f "$HOME/.claude/settings.json" ]; then
+        jq '{enabledPlugins: .enabledPlugins}' "$HOME/.claude/settings.json" \
+            > "$CLAUDE_STAGING/enabled-plugins.json" 2>/dev/null || true
+    fi
+
+    # Agents & skills: copy only what the user listed in .env
+    CLAUDE_AGENTS=""
+    CLAUDE_SKILLS=""
+    ENV_FILE=".devcontainer/.env"
+    [ -f "$ENV_FILE" ] && source "$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)